components/calendar/calendar.component.ts

```diff
@@ -457,6 +468,7 @@ export class CalendarComponent implements ControlValueAccessor, OnInit, OnDestro
   }
 
   ngOnDestroy() {
+    this._visibleChanges$.complete();
```

Comment:
Here you need to put 
this._visibleChanges$.next();
this._visibleChanges$.complete();

takeUntil takes next as emission. If only complete() called it won't unsubscribe
https://cartant.medium.com/rxjs-avoiding-takeuntil-leaks-fb5182d047ef

Reply:
In this lib when it's a boolean they tries to respect the naming convention **isBlabla**
Your variable does not respect it

Reply:
Other question: why have you created:  **_visibleChanges** ? 
Here you can use directly **_unsubscribe**

Reply:
I don't subscribe to anything. `_visibleChanges` is here to cancel if necessary the observable created when visible is set to `false` (which delays the setting of `isHidden`).